### PR TITLE
fixed PHOENIX A0V atmosphere read error & identify save error

### PR DESCRIPTION
**Pull request by Adolfo1519**

### CODE REVIEW

File: pypeit/core/gui/identify.py [hunk 1]
```diff
@@ -870,43 +870,43 @@ def store_solution(self, final_fit, binspec, rmstol=0.15,
...
+                        wvcalib.to_file()
+                        if multi:
+                            slit_list_str = ''; slit_list = np.arange(np.shape(specdata)[0])
+                            for islit in slit_list:
+                                if islit < len(slit_list) - 1:
+                                    slit_list_str += str(islit) + ','
+                                else: slit_list_str += str(islit)
+
+                            if slits:
+                                print('               ')
```
Comment (VascoSch92) [inline pypeit/core/gui/identify.py]:
```suggestion
                                print('  '*10)
```
it is a little bit better to read :-)

Comment (Adolfo1519) [reply pypeit/core/gui/identify.py]: Thank you!

File: pypeit/core/gui/identify.py [hunk 2]
```diff
@@ -870,43 +871,43 @@ def store_solution(self, final_fit, binspec, rmstol=0.15,
...
+                            msgs.warn('Do you want to overwrite existing Calibrations/WaveCalib*.fits file? ' + msgs.newline() +
+                                    'NOTE: To use this WaveCalib file the user will need to delete the other files in Calibrations/ ' + msgs.newline() +
+                                    ' and re-run run_pypeit. ')
+                            print('')
+                            ow_wvcalib = input('Proceed with overwrite? (y/[n]): ')
+
+                    if ow_wvcalib == 'y':
+                        wvcalib.to_file()
+                        if multi:
+                            slit_list_str = ''; slit_list = np.arange(np.shape(specdata_multi)[0])
```
Comment (rcooke-ast) [inline pypeit/core/gui/identify.py]: For readability, please write this on two seaprate lines
```
slit_list_str = ''
slit_list = np.arange(np.shape(specdata_multi)[0])
```

**Review by rcooke-ast — APPROVED** [review]
Looks good to me - thanks for implementing the fix!

File: pypeit/core/flux_calib.py [hunk 3]
```diff
@@ -431,7 +431,7 @@ def get_standard_spectrum(star_type=None, star_mag=None, ra=None, dec=None):
         elif 'PHOENIX' in star_type:
             msgs.info('Getting PHOENIX 10000 K, logg = 4.0 spectrum')
             ## Vega model from TSPECTOOL
-            vega_file = data.Paths.standards / 'PHOENIX_10000K_4p0.fits'
+            vega_file = dataPaths.standards.get_file_path('PHOENIX_10000K_4p0.fits')
```
Comment (kbwestfall) [inline pypeit/core/flux_calib.py:434]: Nice!

File: pypeit/core/gui/identify.py (hunk 2 above)
Comment (kbwestfall) [reply pypeit/core/gui/identify.py]: Yes, please!

**Review by kbwestfall — CHANGES_REQUESTED** [review]
Looks good to me.  Please add a comment to the `doc/releases/1.16.1dev.rst` file.

File: pypeit/core/gui/identify.py (hunk 2 above)
Comment (Adolfo1519) [reply pypeit/core/gui/identify.py]: done!

File: pypeit/core/flux_calib.py (hunk 3 above)
Comment (Adolfo1519) [reply pypeit/core/flux_calib.py:434]: Thanks!

**Review by kbwestfall — APPROVED** [review]
Thanks for the update to the release doc!

**Pull request merged**